hugo/layouts/partials/head.html

```diff
@@ -12,9 +12,13 @@
   {{ end }}
   <!-- Site Title, Description and Favicon -->
 
-  {{- with .Title | default .Site.Title }}
-  <title>{{ . }} | Shine</title>
-  {{- end }}
+  {{ if .Params.metaTitle }}
```

Comment:
Will this code change the title used in tab? 
![Screen Shot 2019-12-17 at 10 48 07 AM](https://user-images.githubusercontent.com/15316174/71011038-bc194500-20ba-11ea-9785-74f8f2da0873.png)

Reply:
Is that intentional?

Reply:
Yea, it changes what's shown both in the tab title and in the title shown in Google Search results.

The `og:title` and `twitter:title` should change what's shown in the the previews for their respective platforms.

Basically this is giving them the ability to set a different external-facing title to catch eyes vs. what we show on the article page itself to readers.

Reply:
got it. I wasn't sure if the meta title was only for searching or if we wanted to display it anywhere in the browser.